src/main/java/duke/Pikachu.java

```diff
@@ -0,0 +1,247 @@
+package duke;
+import java.io.*;
+import java.time.*;
+import java.util.*;
+
+import duke.UI.UI;
+import duke.tasks.*;
+
+
+public class Pikachu {
+    private ArrayList<Task> inputList;
+
+    //Constructor
+    public Pikachu() {
+        inputList = new ArrayList<Task>();
+    }
+
+    //Constructor for existing tasklist
+    public Pikachu(ArrayList<Task> inputList) {
+        this.inputList = inputList;
+    }
+
+    //Accessor
+    public ArrayList<Task> getInputList() {
+        return this.inputList;
+    }
+
+    /**
+     * Parses input from the user. This is the main command for Pikachu.
+     *
+     * @param str Input from user.
+     */
+    public void parseInput(String str) {
+        String[] split = str.split(" ");
```

Comment:
Should variable name for arrays be singular or plural? Similar issues throughout this function.
```suggestion
        String[] splitStrings = str.split(" ");
```